sdk/communication/azure-communication-chat/azure/communication/chat/_models.py

```diff
@@ -228,7 +231,8 @@ def _from_generated(cls, chat_thread):
             id=chat_thread.id,
             topic=chat_thread.topic,
             created_on=chat_thread.created_on,
-            created_by=created_by
+            created_by=created_by,
+            retention_policy=chat_thread.retention_policy
```

Comment:
This will be returning the generated model rather than the SDK model.
```suggestion
            retention_policy=ChatRetentionPolicy(
                delete_thread_after_days=chat_thread.retention_policy.delete_thread_after_days
            )
```

Reply:
This will need to be updated for the ChatThreadItem result as well

Reply:
This still needs to be resolved for ChatThreadItem